.github/workflows/ci.yml

```diff
@@ -101,6 +101,43 @@ jobs:
     needs: run_rubocop
     runs-on: ubuntu-22.04
     services:
+      elasticsearch7:
+        image: elasticsearch:7.16.2
+        env:
+          discovery.type: single-node
+        ports:
+          - 9200:9200
+        options: >-
+          --health-cmd "curl http://localhost:9200/_cluster/health"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 10
+      elasticsearch8:
+        image: elasticsearch:8.4.2
+        env:
+          discovery.type: single-node
+          xpack.security.enabled: false
+          ES_JAVA_OPTS: "-Xms1g -Xmx1g"
```

Comment:
I'm curious about this one.

- I'm not sure where we tell GHA that we'd like to have a gig of memory available to be allocated here
- My hunch is that in a docker-compose / GHA services context, the `java` process has already been started and this env var might be getting ignored.

Reply:
Yeah, I'm not sure it's really needed here. I included it with 8 here because of an example for 8 I found when I was figuring it out. But after working on the issues with docker compose I ran into with docker running out of memory, and since the CI doesn't ever have that problem, this is probably either ignored or pointless in a GHA context.